### DISCUSSION TITLE
How to receive the result of transaction

### DISCUSSION THREAD

**Discussion by yjk596286200** [ton-org/game-engines-sdk#16] [Q&A]
Hi, I'm having some trouble and would love to get some help.My question is after transferTon(), how do I resieve the result or transaction response. I can find the resut in console.My code is here:
` const tonTransferReq = {
                to: Address.parse(this._tonAddressConfig.tonAddress),
                amount: toNano(0.01)
            } as TonTransferRequest;

            try {
                await this._gameFi.transferTon(tonTransferReq);
            } catch (error) {
                console.error(error)
            }
            `

**Comment by barinbritva** [comment]
Hi @yjk596286200 

To see transactions on a wallet you need to request a blockchain lite node. The easiest way to do so is using tonweb API - https://github.com/toncenter/tonweb. You need to use `getTransactions` method for that
